src/components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddContractCallPanel/components/CreateNftForm/hooks/useCreateNft.ts

```diff
@@ -68,9 +68,16 @@ const useCreateNft = (
   const createNft = async (data: CreateNftFormType): Promise<CreateNFTResponse> => {
     setLoadingText("Uploading image")
 
-    const { IpfsHash: imageCID } = await pinFileToIPFS({
-      data: [data.image],
-    })
+    let imageCID
+
+    if (process.env.NEXT_PUBLIC_MOCK_CONNECTOR) {
```

Comment:
Not sure if it is optimal to put testing logic here, but I couldn't came up with a better idea. Any suggestions? 👀

Reply:
Oh, yup, great idea, fixed in b7e8abbeec38958e8df837cb60040b67399a7e65